src/Domain/Repository/BuildingRepositoryInterface.php

```diff
@@ -9,6 +9,6 @@
 
 interface BuildingRepositoryInterface
 {
-    public function add(Building $building);
+    public function store(Building $building);
```

Comment:
This was renamed in order to make it clear that we aren't necessarily creating and persisting **new** instances of `Building`, but that we are simply storing existing state mutations (new events) when necessary.